seahorn/jobs/hash_table_foreach/aws_hash_table_foreach_harness.c

```diff
@@ -0,0 +1,29 @@
+#include <aws/common/hash_table.h>
+#include <aws/common/private/hash_table_impl.h>
+#include <config.h>
+#include <hash_table_helper.h>
+#include <seahorn/seahorn.h>
+#include <utils.h>
+
+int hash_table_foreach_proof_callback(void *context,
+                                      struct aws_hash_element *pElement) {
+  AWS_PRECONDITION(AWS_OBJECT_PTR_IS_WRITABLE(pElement),
+                   "Input pointer [pElement] must be writable.");
+  return nd_int();
+}
+
+int main(void) {
+  struct aws_hash_table map;
+  initialize_bounded_aws_hash_table(&map, MAX_TABLE_SIZE);
+  ensure_hash_table_has_valid_destroy_functions(&map);
+  map.p_impl->equals_fn = nondet_equals;
+  map.p_impl->hash_fn = uninterpreted_hasher;
+  assume(aws_hash_table_is_valid(&map));
+
+  void *context;
```

Comment:
better initialize this pointer. Nothing happens to it, but might as well initialize to something like `nd_voidp()`